tests/udp/test_udp_header_with_fixed_length_checksum_e2e.py

```diff
@@ -3,13 +3,8 @@
 from abstract_open_traffic_generator import flow as Flow
 
 
-@pytest.mark.skip(reason="skip until moved to other repo")
-@pytest.mark.parametrize('packets', [1000])
```

Comment:
I prefer the packets and size to be parametrize for e2e tests, would be easy to increase test coverage if more unit test is required. But if tests to be look consistent, this is also fine for me

Reply:
I agree with parametrizing those variables. The problem is, should we or should we not parametrize other variables in the func as well. We need to converge on following:
- choose general variables that should be parameterized for all tests
- since those variables are pretty common across all tests, is there a way to control the parametrization from somewhere central rather than specifying them in the test script ? (pytest has something similar I guess)

Point is, we want to do it, but probably there's a better way than embedding same snippet in each test script.
Please raise an issue to specifically track this.